src/main/scala/com/fulcrumgenomics/vcf/JointVariantContextIterator.scala

```diff
@@ -55,26 +56,25 @@ class JointVariantContextIterator private(iters: Seq[Iterator[VariantContext]],
                                           dictOrComp: Either[SequenceDictionary, VariantContextComparator]
                                          )
 extends Iterator[Seq[Option[VariantContext]]] {
-  import com.fulcrumgenomics.fasta.Converters.ToSAMSequenceDictionary
 
   if (iters.isEmpty) throw new IllegalArgumentException("No iterators given")
 
   private val iterators = iters.map(_.buffered)
-  private val comparator = dictOrComp match {
-    case Left(dict)  => new VariantContextComparator(dict.asSam)
```

Comment:
The class `VariantContextComparator` now compares more than just on location.

Searching the codebase, this was the only improper use of the new class (and unit tests were failing).